Wikipedia/Code/LoginFunnel.swift

```diff
@@ -17,11 +17,6 @@
     }
     
     private func event(category: EventLoggingCategory, label: EventLoggingLabel?, action: Action, measure: Double? = nil) -> Dictionary<String, Any> {
-        guard category != .undefined else {
-            assertionFailure("category cannot be undefined")
-            return [:]
-        }
```

Comment:
The only `unknown` case that we want to catch will be caught here:

https://github.com/wikimedia/wikipedia-ios/blob/efb079dbddb15f54ded57c40908f91cb9561e355/Wikipedia/Code/ArticleCollectionViewController.swift#L84